opentracing-web-servlet-filter/pom.xml

```diff
@@ -27,10 +27,16 @@
   <properties>
     <version.org.eclipse.jetty>9.3.15.v20161220</version.org.eclipse.jetty>
     <version.com.squareup.okhttp3>3.5.0</version.com.squareup.okhttp3>
+    <version.org.mockito>1.10.19</version.org.mockito>
+    <version.ch.qos.logback>1.1.11</version.ch.qos.logback>
     <main.basedir>${project.basedir}/..</main.basedir>
   </properties>
 
   <dependencies>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-api</artifactId>
```

Comment:
I am not sure if we want to include even this. There should be perhaps a separate MDC instrumentation

Reply:
Yeah I wasn't sure neither. But felt that slf4j-api was pretty minimal. We could split it in aa different Maven module as well